jni/Android.mk

```diff
@@ -78,7 +78,7 @@ else
     modules/lfo.cpp \
     modules/routeableoscillator.cpp \
 
-    LOCAL_LDLIBS := -llog -lOpenSLES
+    LOCAL_LDLIBS := -llog -lOpenSLES -latomic -landroid
 
```

Comment:
Newer NDK version require the -latomic flag -android added to support AAssetManager